simple/provider.go

```diff
@@ -14,7 +15,10 @@ import (
 
 var logP = logging.Logger("provider.simple")
 
-const provideOutgoingWorkerLimit = 8
+const (
+	provideOutgoingWorkerLimit = 8
```

Comment:
I think this doesn't belong here?

Reply:
This does belong here, but I went ahead and made it configurable.

Reply:
Sorry, I think I only saw I subdiff and didn't see the whole thing before.